Changelog.md

```diff
@@ -7,6 +7,7 @@ Compiler Features:
  * Error Reporting: Unimplemented features are now properly reported as errors instead of being handled as if they were bugs.
  * EVM: Support for the EVM version "Prague".
  * SMTChecker: Add CHC engine check for underflow and overflow in unary minus operation.
+ * SMTChecker: Passing all SMT tests now requires Eldarica in addition to Z3.
```

Comment:
@nikola-matic , Compiler Features seems like the most appropriate section for this kind of entry. Feel free to suggest modifications.

Reply:
Yeah, compiler features should be fine.